src/libs/vwidgets/export_format_combobox.cpp

```diff
@@ -0,0 +1,330 @@
+QVector<std::pair<QString, LayoutExportFormat>> ExportFormatCombobox::initFormats()
+{
+    QVector<std::pair<QString, LayoutExportFormat>> list;
+
+    auto InitFormat = [&list](LayoutExportFormat format)
+    {
+        list.append(std::make_pair(exportFormatDescription(format), format));
+    };
+
+    InitFormat(LayoutExportFormat::SVG);
+    InitFormat(LayoutExportFormat::PDF);
+    InitFormat(LayoutExportFormat::PDFTiled);
+    InitFormat(LayoutExportFormat::PNG);
+    InitFormat(LayoutExportFormat::JPG);
+    InitFormat(LayoutExportFormat::BMP);
+    InitFormat(LayoutExportFormat::TIF);
+    InitFormat(LayoutExportFormat::PPM);
+    InitFormat(LayoutExportFormat::OBJ);
+    if (supportPSTest())
+    {
+        InitFormat(LayoutExportFormat::PS);
+        InitFormat(LayoutExportFormat::EPS);
+    }
+    InitFormat(LayoutExportFormat::DXF_AC1006_Flat);
+    InitFormat(LayoutExportFormat::DXF_AC1009_Flat);
+    InitFormat(LayoutExportFormat::DXF_AC1012_Flat);
+    InitFormat(LayoutExportFormat::DXF_AC1014_Flat);
+    InitFormat(LayoutExportFormat::DXF_AC1015_Flat);
+    InitFormat(LayoutExportFormat::DXF_AC1018_Flat);
+    InitFormat(LayoutExportFormat::DXF_AC1021_Flat);
+    InitFormat(LayoutExportFormat::DXF_AC1024_Flat);
+    InitFormat(LayoutExportFormat::DXF_AC1027_Flat);
+    InitFormat(LayoutExportFormat::DXF_AC1006_AAMA);
+    InitFormat(LayoutExportFormat::DXF_AC1009_AAMA);
+    InitFormat(LayoutExportFormat::DXF_AC1012_AAMA);
+    InitFormat(LayoutExportFormat::DXF_AC1014_AAMA);
+    InitFormat(LayoutExportFormat::DXF_AC1015_AAMA);
+    InitFormat(LayoutExportFormat::DXF_AC1018_AAMA);
+    InitFormat(LayoutExportFormat::DXF_AC1021_AAMA);
+    InitFormat(LayoutExportFormat::DXF_AC1024_AAMA);
+    InitFormat(LayoutExportFormat::DXF_AC1027_AAMA);
+    // We will support them anyway
+//    InitFormat(LayoutExportFormat::DXF_AC1006_ASTM);
+//    InitFormat(LayoutExportFormat::DXF_AC1009_ASTM);
+//    InitFormat(LayoutExportFormat::DXF_AC1012_ASTM);
+//    InitFormat(LayoutExportFormat::DXF_AC1014_ASTM);
+//    InitFormat(LayoutExportFormat::DXF_AC1015_ASTM);
+//    InitFormat(LayoutExportFormat::DXF_AC1018_ASTM);
+//    InitFormat(LayoutExportFormat::DXF_AC1021_ASTM);
+//    InitFormat(LayoutExportFormat::DXF_AC1024_ASTM);
+//    InitFormat(LayoutExportFormat::DXF_AC1027_ASTM);
+
+    return list;
+}
+
+//---------------------------------------------------------------------------------------------------------------------
+bool ExportFormatCombobox::supportPSTest()
+{
+    if (!tested)
+    {
+        havePdf = testPdf();
+        tested = true;
+    }
+    return havePdf;
+}
+
+//---------------------------------------------------------------------------------------------------------------------
+bool ExportFormatCombobox::testPdf()
+{
+    bool res = false;
+
+    QProcess proc;
+    QStringList args;
+
+#if defined(Q_OS_WIN) || defined(Q_OS_OSX)
+    // Seek pdftops in app bundle or near valentin.exe
```

Comment:
? valentin.exe ?

Reply:
I assume this was meant to be Valentina exe... I didn't put the comment there - Since I wanted to use an export format combo box in more than the one place, I subclassed combobox and moved some of the code there. I wouldn't doubt if there's a few more Valentin.exe's hiding.

Reply:
I left the com eny, but just removed the reference to Valentin.exe